app/models/claim/base_claim.rb

```diff
@@ -329,6 +329,10 @@ def earliest_representation_order_date
       earliest_representation_order.try(:representation_order_date)
     end
 
+    def clar?
+      earliest_representation_order_date >= Settings.clar_release_date.to_date.beginning_of_day
+    end
```

Comment:
While this new helper is fine of itself we need to check for potential routes through the app where rep order is unknown. 

In particular have a look at `FeeScheme.for_claim` and `FeeScheme#agfs_scheme_12?` methods

Reply:
Look into handling nil rep order dates with a sensible default - false. add spec for it too.

Reply:
I have a test that engineers a `nil` value for `earliest_representation_order_date` and now that will return false for `clar?`. It would be helpful to review this to see if it makes sense.